src/main/java/pl/sdacademy/pewex/db/ClientEntity.java

```diff
@@ -0,0 +1,87 @@
+package pl.sdacademy.pewex.db;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "clients")
+public class ClientEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(length = 150, unique = true)
+    private String userName;
+
+    @Column(length = 150, unique = true)
+    private String password;
+
+    @Column(length = 150, unique = true)
+    private String role;
+
+    @Column(length = 150, unique = true)
```

Comment:
A tutaj to grubo jest :) 
Taka adnotacja dla klucza do innej tabeli?
Czemu nagle int jak wyżej id jest jako long?

Reply:
zmieniono